### ISSUE TITLE
Allow either player to break the follow-up game chain

### ISSUE THREAD

**Issue by blackshadowshade** [buttonmen-dev/buttonmen#1831]
In game 8649 (http://www.buttonweavers.com/ui/game.html?game=8649), irilyth has created a follow-up game from game 8439 (http://www.buttonweavers.com/ui/game.html?game=8439), I presume because it simplified the game creation process. However, it is aesthetically messy to have two separate stories linked in a chain of follow-up games.

It should be possible for either player to unlink the game from its previous game so that the chain is broken.

**Comment by cgolubi1** [comment]
An alternative would be to only allow chat to be continued from a given game once (e.g. the function that tests if a continuation is valid could look for other non-rejected games which continue chat from the first game, and return invalid if one is found).  Which would be better for your use case?

**Comment by blackshadowshade** [comment]
My current use case is the haruspex vs haruspex story game, where any continuation is probably undesirable.

**Comment by irilyth** [comment]
I'm not at all convinced that continuation is undesirable. I think it's nice to be able to go back through past stories. What don't you like about it? "Aesthetically messy"?

**Comment by blackshadowshade** [comment]
I dislike seeing multiple stories in the chat log when each story is ideally standalone.

**Comment by irilyth** [comment]
Mm. That gets back to some of the recent talk about chat logs and chronological order and stuff. For one thing, completed games already only show the chat for that one game, right? e.g. http://www.buttonweavers.com/ui/game.html?game=8403

I feel like a solution that let *each* player hide the chat from the previous game, if they want to, would be better than a per-game or site-wide setting about this. What if, at the "Continued from Game 8439" button, there was a "hide previous chat" button? (Ideally this would persist for you for that game every time you loaded it, but that might be hard, dunno.)

**Comment by blackshadowshade** [comment]
I'm thinking from the point of view of someone kibitzing. I can see the potential for confusion when a kibitzer is reading through a story, and then there's the dislocation into the next story. Sure, there's the visual cue of the "Continued from ...", but it doesn't really stand out.

**Comment by blackshadowshade** [comment]
I take your point that completed games don't show chat from previous rounds, I just wish that an active game also had the same possibility of being structurally disjoint in its chat from its predecessor.

**Comment by irilyth** [comment]
Yeah, I don't know how often people watch other people's games in progress -- I'll read stories from completed games sometimes, but those only show the one game's story anyway. (And when they do it in chronological order, *that'll* be a big improvement.) If you (or others) do, maybe it's worth doing something there.

Mostly I don't want to lose the chain -- I don't actually care so much about the previous story (or the previous chat in general) being displayed.

I'm also generally wary of "either player should be able to make a UI decision that affects both players", especially when it's changing something that one player is already looking at. For example, one player gets to decide what the game description is, but (a) that's hard to avoid; (b) that happens before the other player sees the description, so it's not too weird; (c) there's already this model that the creator of the game gets to set the description, so even if the creator can change it after the fact, that seems less disruptive than letting me make your previous chat history disappear.

**Comment by irilyth** [comment]
> I'm also generally wary of "either player should be able to make a UI decision that affects both players"

Another example of this: I can choose what color my mat is when I see it; should I be able to choose what color my mat is when you see it? If we were playing in person, I'd get to choose what color shirt I was wearing. :^) But I think it's better in this case if you have control over your viewing experience, even if what you're viewing is some representation of "me".

**Comment by blackshadowshade** [comment]
If I weren't accepting all games automatically, I would have had the chance to reject your proposed game and counter with a non-linked game with the same buttons.

I don't see how unlinking games is any different to doing this.
